frontend2/src/pages/prompt/PromptEntryPage.tsx

```
@@ -0,0 +1,62 @@
+  const isValidWorkspaceId = Number.isInteger(parsedWorkspaceId) && parsedWorkspaceId > 0;
+  const navigate = useNavigate();
+
+  if (!isValidWorkspaceId) {
+    return <div className="p-8 text-gray-500">유효하지 않은 워크스페이스입니다.</div>;
+  }
+
+  const workspaceId = parsedWorkspaceId;
+  const basePath = orgId ? `/orgs/${orgId}/workspaces/${workspaceId}` : `/workspaces/${workspaceId}`;
+
+  const { data: prompts, isLoading } = useQuery({
+        queryKey: ['prompts', workspaceId],
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)